src/FilterBundle/Manager/SuggestManager.php

```diff
@@ -187,10 +242,14 @@ public function getProducts(string $keyword, int $max): array
         }
 
         $products = [];
-        foreach ($response->toObject() as $product) {
+        foreach ( $response->toObject() as $product) {
             $products[] = $this->getProductNormalizer()->normalize($product);
```

Comment:
Ich muss gestehen, dass ich in diesem Bundle auf keinen Fall mit einem Normalizier arbeiten würde. Warum auch?

Reply:
Der kleinste Nenner, PHP SDK --> original Models, wäre besser für eine Lib deutlich gesünder.

Reply:
Ja stimme ich dir zu. Allerdings war dieses Bundle ja zu Anfangs keine "richtige" Library. Der Code war noch Projektbezogen und wurde später erst mit den zur Verfügung stehenden Ressourcen "herausgezogen".